src/elasticodbc/dlg_specific.h

```diff
@@ -178,6 +186,12 @@ struct authmode {
     const char *authtype_str;
 };
 const struct authmode *GetAuthModes();
+
+struct loglevel {
+    int loglevel_id;
+    const char *loglevel_str;
```

Comment:
I dont think this str member is being used - you can probably remove this and remove the LOGTYPE_* defines usage.

Reply:
removed